corehq/apps/locations/views.py

```diff
@@ -360,6 +360,20 @@ def archive_location(request, domain, loc_id):
         )
     })
 
+@can_edit_location
+def delete_location(request, domain, loc_id):
```

Comment:
Does this need `method.request` check? Otherwise seems like GET to /<loc_id>/delete might be able to delete too?